### PR TITLE
Fix linting errors, add AUTHORS metadata, update documentation

### PR DESCRIPTION
**Pull request by GiorgiBeriashvili**
## Description

An attempt to improve package's documentation.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# This is the list of http-mock-adapter authors for copyright purposes.
+#
+# This does not necessarily list everyone who has contributed code, since in
+# some cases, their employer may be the copyright holder. To see the full list
+# of contributors, see the revision history in source control.
+Lomsa
+Dato Barbakadze
+Giorgi Beriashvili
+Luka Giorgadze
+Natalie Alibegashvili
```
Comment (LukaGiorgadze) [inline AUTHORS:10]: ხომ არ ჯობია აქ სახელის კლიკზე გითაბის ექაუნთზე გადავიდეს?

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,3 @@
+* text=auto eol=lf
+*.{cmd,[cC][mM][dD]} text eol=crlf
+*.{bat,[bB][aA][tT]} text eol=crlf
```
Comment (LukaGiorgadze) [inline .gitattributes:3]: i would prefer LF

ref: https://stackoverflow.com/questions/1552749/difference-between-cr-lf-lf-and-cr-line-break-types

**Review by LukaGiorgadze — CHANGES_REQUESTED** [review]

File: AUTHORS (hunk 1 above)
Comment (GiorgiBeriashvili) [reply AUTHORS:10]: I don't know, this is is an informal standard used throughout many GitHub projects. It's not a markdown file.

Many open-source projects write name and surname and be done with it. Some [also add emails for contacting purposes](https://github.com/flutter/flutter/blob/master/AUTHORS).

File: .gitattributes (hunk 2 above)
Comment (GiorgiBeriashvili) [reply .gitattributes:3]: Why?

It's written that only `cmd` and `bat` files will have `crlf` line ending. Everything else will have `lf` line ending.

`cmd` and `bat` are included for compatibility purposes (specifically for Windows) if there ever will be the need.

File: AUTHORS (hunk 1 above)
Comment (LukaGiorgadze) [reply AUTHORS:10]: Ah, okay got it. You are good man.

File: .gitattributes (hunk 2 above)
Comment (LukaGiorgadze) [reply .gitattributes:3]: Oh, i just noticed cmd & bat only files. Then it's good.

**Review by LukaGiorgadze — APPROVED** [review]

File: AUTHORS (hunk 1 above)
Comment (GiorgiBeriashvili) [reply AUTHORS:10]: We could specify the GitHub links in another way, most probably inside the **README.md** file.

Comment (GiorgiBeriashvili) [reply AUTHORS:10]: @LukaGiorgadze should I include the emails inside **AUTHORS**? and GitHub links inside the **README.md**?

Comment (LukaGiorgadze) [reply AUTHORS:10]: No, i guess full name is enough for now. Everyone can see contributors here.

**Review by nataliehippe — APPROVED** [review]

**Review by campfire5 — APPROVED** [review]

**Pull request merged**